docker/release/opensuse13.2/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Docker image file that describes an OpenSUSE 13.2 (AKA harlequin) image with PowerShell installed from PowerShell Release
+FROM opensuse:harlequin
+LABEL maintaner Travis Plunk <travis.plunk@microsoft.com>
```

Comment:
from https://github.com/docker/docker/pull/25466/files


+The `MAINTAINER` instruction sets the *Author* field of the generated images. 
 +The `LABEL` instruction is a much more flexible version of this and you should use
 +it instead, as it enables setting any metadata you require, and can be viewed
 +easily, for example with `docker inspect`. To set a label corresponding to the
 +`MAINTAINER` field you could use:
 +
 +    LABEL maintainer "SvenDowideit@home.org.au"